astropy/time/formats.py

```diff
@@ -710,38 +710,45 @@ class TimeUnix(TimeFromEpoch):
 
 class TimeUnixTai(TimeUnix):
     """
-    Seconds from 1970-01-01 00:00:08 TAI (see notes), including leap seconds.
+    Unix time (TAI): SI seconds elapsed since 1970-01-01 00:00:00 TAI (see caveats).
 
-    This will generally differ from Unix time by the cumulative integral number
-    of leap seconds since 1970-01-01 UTC.  This convention matches the definition
-    for linux CLOCK_TAI (https://www.cl.cam.ac.uk/~mgk25/posix-clocks.html).
+    This will generally differ from standard (UTC) Unix time by the cumulative
+    integral number of leap seconds introduced into UTC since 1970-01-01 UTC.
```

Comment:
The sentence is missing the `plus the initial offset of 8 seconds`

Reply:
The correct sentence would be 

* This will generally differ from standard (UTC) Unix time by the cumulative
   integral number of leap seconds introduced into UTC since 1972-01-01 UTC plus the initial offset of 10 seconds at that date. 
* Between 1970-01-01 and 1972-01-01 there were also fractional second adjustments made to UTC, resulting in an offset of `8.000082` seconds at the unix epoch of `1970-01-01 00:00:00 UTC`.

Reply:
> . There have been 29 leap seconds since 1970, not 37.

No, 27 leap seconds. The additional 2 seconds come from fractional adjustments between 1970 and 1972.

Reply:
updated, please have another look